.github/workflows/go.yml

```diff
@@ -36,3 +36,18 @@ jobs:
     - name: Check tidiness
       run: |
         ./ci/check-tidy.sh
+
+  golangci:
+    name: lint
+    runs-on: [ubuntu-latest]
+    steps:
+    - uses: actions/checkout@v2
+    - name: golangci-lint
+      uses: golangci/golangci-lint-action@v2
+      with:
+        # Required: the version of golangci-lint is required and must be specified without patch version:
+        # we always use the latest patch version.
+        version: v1.39
```

Comment:
maybe add a comment that this needs to be kept in sync with the version in the Makefile